tests/console/yast2_apparmor.pm

```diff
@@ -0,0 +1,143 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+
+
+sub run() {
+
+    select_console 'root-console';
+
+    # install yast2_apparmor package at first
+    assert_script_run("/usr/bin/zypper -n -q in yast2-apparmor");
+
+    # start apparmor configuration
+    script_run("/sbin/yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
+
+    # check Apparmor Configuration is opened
+    assert_screen 'yast2_apparmor';
+    send_key 'ret';
+    assert_screen 'yast2_apparmor_enabled';
+
+    # part 1: open profile mode configuration and check toggle/show all profiles
+    send_key 'alt-n';
+    assert_screen 'yast2_apparmor_profile_mode_configuration';
+    send_key 'alt-o';
+    wait_still_screen;
```

Comment:
`wait_still_screen` by default waits for 7 seconds. Does it also work with less? E.g. 1 or 2 seconds? (that would be `wait_still_screen(1);`